packages/login/theme/material/vaadin-login-form.js

```diff
@@ -1,3 +1,4 @@
+import '@vaadin/button/theme/material/vaadin-button.js';
 import '@vaadin/text-field/theme/material/vaadin-text-field.js';
 import '@vaadin/password-field/theme/material/vaadin-password-field.js';
 import './vaadin-login-form-wrapper.js';
```

Comment:
Should we also move imports from Material's `vaadin-login-from-wrapper.js` directly to this file, like it was done for the Lumo theme?

Reply:
Thanks, done.